File: programming_paradigms/object_oriented_programming/java_examples/ObjectOrientedEasy.java
package programming_paradigms.object_oriented_programming.java_examples;

class Car {
    String brand;
    String model;
    int year;

    Car(String brand, String model, int year) {
        this.brand = brand;
        this.model = model;
        this.year = year;
    }

    void displayInfo() {
        System.out.println("Brand: " + brand + ", Model: " + model + ", Year: " + year);
    }
}

public class ObjectOrientedEasy {
    public static void main(String[] args) {
        Car car = new Car("Toyota", "Camry", 2022);
        car.displayInfo();
    }
}

File: programming_paradigms/object_oriented_programming/golang_examples/ObjectOrientedEasy.go
package golangexamples

import "fmt"

type Car struct {
	Brand string
	Model string
	Year  int
}

func (c Car) DisplayInfo() {
	fmt.Printf("Brand: %s, Model: %s, Year: %d\n", c.Brand, c.Model, c.Year)
}

func main() {
	car := Car{Brand: "Toyota", Model: "Camry", Year: 2022}
	car.DisplayInfo()
}
